src/akeneo/harness.yml

```diff
@@ -134,11 +134,6 @@ attributes:
     image:
       console: = 'my127/akeneo:' ~ @('php.version') ~ '-fpm-buster-console'
       php-fpm: = 'my127/akeneo:' ~ @('php.version') ~ '-fpm-buster'
-  elasticsearch:
-    image: elasticsearch
-    tag: '7.10.1'
-  mysql:
-    tag: '8.0'
```

Comment:
huh?

Reply:
it's 8.0 in base

Reply:
Ah is the default 8.0 now?

Reply:
yep, so devs don't suffer the performance penalty of mysql/mysql-server also not supporting arm64 (though it handles the platform choice better than mysql:8.0 insisting on platform being hard-coded